test/integration/rhel-8.3/gcc-static-local-var-6.patch

```diff
@@ -0,0 +1,20 @@
+diff -Nupr src.orig/net/ipv6/netfilter.c src/net/ipv6/netfilter.c
+--- src.orig/net/ipv6/netfilter.c	2020-11-30 14:09:23.759997161 -0500
++++ src/net/ipv6/netfilter.c	2020-11-30 14:12:24.466069511 -0500
+@@ -71,11 +71,16 @@ int ip6_route_me_harder(struct net *net,
+ }
+ EXPORT_SYMBOL(ip6_route_me_harder);
+ 
++#include "kpatch-macros.h"
++
+ static int nf_ip6_reroute(struct sk_buff *skb,
+ 			  const struct nf_queue_entry *entry)
+ {
+ 	struct ip6_rt_info *rt_info = nf_queue_entry_reroute(entry);
+ 
++	if (!jiffies)
++		printk("kpatch nf_ip6_reroute foo\n");
++
```

Comment:
I think this printk() should be in __nf_ip6_route() instead of nf_ip6_reroute().  8.3's __nf_ip6_route() is a renamed version of 8.2's nf_ip6_route().  (Also based on the name of the test it probably tests a change to a function which has a static local variable, which __nf_ip6_route() does).

Reply:
This is updated in the squashed version I am pushing. I realized that I made that change too quick.

Thanks for confirming.